kratos/sources/kratos_parameters.cpp

```diff
@@ -581,6 +581,24 @@ bool Parameters::IsArray() const
 /***********************************************************************************/
 /***********************************************************************************/
 
+bool Parameters::IsStringArray() const
+{
+    if (!mpValue->is_array()) {
+        return false;
+    } else {
+        const auto& r_array = (*mpValue);
+        for (IndexType i = 0; i < mpValue->size(); ++i) {
+            if (!r_array[i].is_string()) {
+                return false;
+            }
+        }
+        return true; // All entries are numbers or Vector is empty
```

Comment:
```suggestion
        return true; // All entries are strings or Vector is empty
```

Reply:
Sorry, I have seen that the PR was approved and I merged it before you added the comments... Should I open another PR to apply your suggestion?

Reply:
Yes, lets do it like this. You can even use this very same branch if you want.

Reply:
I have changed the comment in #11200